llvm/test/MC/AArch64/FP8/directive-arch.s

```diff
@@ -3,11 +3,19 @@
 .arch armv9-a+fp8
 bf1cvtl v0.8h, v0.8b
 // CHECK: bf1cvtl v0.8h, v0.8b
-
 .arch armv9-a+nofp8
+
```

Comment:
Why you changed these lines

Reply:
Because the line of .arch armv9-a+nofp8 is related to the code snippet above.